### DISCUSSION TITLE
Pitea MHS sample set

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#491] [Help]
Originally posted 2016-12-28 20:50:28.419000 by piter2015 in GrandOrgue discussion forum on SourceForge<br><br>Hi everyone!
I&#039;m using GO for a couple of months and I&#039;ve just noticed that sampleset from Lars Palo &quot;Pitea MHS&quot; isn&#039;t &quot;complete&quot;. (I know, that this isn&#039;t GO problem, but many of you probably use this sampleset.) Some voices, such us Cornet V or Voix Celeste aren&#039;t full-sacle voices. In the .odf file there is: *NumberOfLogicalPipes=44
NumberOfAccessiblePipes=44*
So, I think, this isn&#039;t problem of my computer setup or something like this...
Is that supposed to be like this?
Thank you

**Comment by larspalo** [comment]
Originally posted 2016-12-28 21:29:08.845000 by [Lars Palo](https://github.com/larspalo) in GrandOrgue discussion forum on SourceForge<br><br>Hi!
Yes, the Voix Celeste and the Cornet V is not &quot;complete&quot; and that&#039;s an original feature of the organ and perfectly in line with the French organ tradition where such stops orignates from.
Actually, you&#039;re in the French tradition never supposed to use the Voix Celeste alone but always together with another string stop (in this case the Gamba). This is in contrast to the english/american tradition of (sometimes) making the stop with two ranks in itself where one is slightly detuned.
The grand cornet is a descant stop and is used either as a solo stop or many times together with the reeds in the French &quot;Grand Jeu&quot; registration (as opposed to the &quot;Plein jeu&quot; with flue stops and mixtures). You can also use it in dialogue pieces in the right hand against the cornet compose (or &quot;Jeu de Tierce du Positif&quot;) on the positif (Ged 8, Fl 4, Kv 2 2/3, Fl 2, Ters 1 3/5) in the left hand (also it&#039;s perfectly possible to add the Princ 4 and Cromorne 8).
So, in short, yes - it&#039;s supposed to be like that!<br><br>Attachments in original post:<br>https://sourceforge.net/p/ourorgan/discussion/962125/thread/0f7bf845/be7a/aa7e/attachment/alternate<br>
